hcx-apis/src/main/java/org/swasth/hcx/controllers/BaseController.java

```diff
@@ -235,11 +259,11 @@ protected void loadNotifications() throws IOException {
         notificationList =  (List<Notification>) jsonReader.readValue(resource.getInputStream(), List.class);
     }
 
-    protected boolean isValidNotificaitonId(String notificationId){
+    protected boolean isValidNotificationId(String notificationId){
```

Comment:
BaseController should have only common logic. Why do we have notification-specific logic here?

Reply:
Moved notification related logic to the notification service.